.eslintrc.js

```diff
@@ -42,6 +42,13 @@ module.exports = {
 		'@wordpress/dependency-group': 'error',
 		'@wordpress/gutenberg-phase': 'error',
 		'@wordpress/react-no-unsafe-timeout': 'error',
+		'@wordpress/valid-text-domain': [
+			'error',
+			{
+				allowDefault: true,
+			},
+		],
+		'@wordpress/no-missing-translator-comments': 'error',
```

Comment:
This line might be unnecessary as it should be part of the recommended rule set already.